selfdrive/car/__init__.py

```diff
@@ -1,8 +1,36 @@
 # functions common among cars
 from common.numpy_fast import clip
-
+from selfdrive.config import Conversions as CV
```

Comment:
can we remove this import? Just so we avoid risk of mutual imports (__int__.py should be very little dependent on other files)